client/src/lib.rs

```diff
@@ -23,7 +21,7 @@ use sp_runtime::{generic,
 	},
 	RuntimeDebug,
 };
-use sp_runtime::traits::{BlakeTwo256, Block};
+use sp_runtime::traits::{BlakeTwo256};
 use sp_std::{collections::vec_deque::VecDeque, prelude::*, str};
 
```

Comment:
Please run `cargo clippy` which points out some redundant `clone()` and other issues.